src/app/content/components/ContentExcerpt.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import { routes } from '..';
+import createTestStore from '../../../test/createTestStore';
+import { book } from '../../../test/mocks/archiveLoader';
+import { mockCmsBook } from '../../../test/mocks/osWebLoader';
+import MessageProvider from '../../MessageProvider';
+import { Store } from '../../types';
+import { receiveBook } from '../actions';
+import { formatBookData } from '../utils';
+import ContentExcerpt from './ContentExcerpt';
+
+describe('ContentExcerpt', () => {
+  let store: Store;
+  let consoleError: jest.SpyInstance;
+
+  beforeEach(() => {
+    store = createTestStore();
+    consoleError = jest
```

Comment:
this is to ignore the error boundary warning? it would be better to add that to the block list in tests/setup so that any other errors are not hidden